### PR TITLE
Add two new solves for syracuse_max()

### PR DESCRIPTION
**Pull request by vveessdv**
* Second solve for 'syracuse_max' func
* Third solve for 'syracuse_max' func
* Final version

### CODE REVIEW

File: pr_4.py [hunk 1]
```diff
@@ -38,12 +38,12 @@ def syracuse_sequence(n):
...
     """
     next_n_func = lambda a: a // 2 if a % 2 == 0 else a * 3 + 1
-    if next_n_func(n)==1:
+    if next_n_func(n) == 1:
         return [n, 1]
     return [n, *syracuse_sequence(next_n_func(n))]
 
 
-def syracuse_max(n):
+def syracuse_max1(n):
```
Comment (Veselovskiydv) [inline pr_4.py:46]: come it back how it was

Comment (Veselovskiydv) [inline pr_4.py]: come back, how it was

File: pr_4.py [hunk 2]
```diff
@@ -56,7 +56,28 @@ def syracuse_max(n):
     next_n_func = lambda a: a // 2 if a % 2 == 0 else a * 3 + 1
     if next_n_func(n) == 1:
         return n
-    return max_func(n, syracuse_max(next_n_func(n)))
+    return max_func(n, syracuse_max1(next_n_func(n)))
```
Comment (Veselovskiydv) [inline pr_4.py]: come back this line too

File: pr_4.py [hunk 3]
```diff
@@ -66,9 +87,15 @@ def main():
     mas = syracuse_sequence(N)
     print(mas)
 
-    max_el = syracuse_max(N)
+    max_el = syracuse_max1(N)
```
Comment (Veselovskiydv) [inline pr_4.py]: and this

**Comment by Veselovskiydv** [comment]
If you make this changes, I'll be happy to merge this.

**Comment by vveessdv** [comment]
Ok,  I will do this 👌

**Pull request merged**